CONTRIBUTING.md

```diff
@@ -15,7 +15,9 @@ any additional information for contributing code to this project.
 
 To release a new version of this chart:
 - Make the appropriate changes
-- Update the version number
+- Update the version number in conjur-oss/Chart.yaml
```

Comment:
Wrap the path in backticks and make it a link to the file for easier finding

Reply:
Fixed.